lib/node_modules/@stdlib/assert/deep-has-own-property/lib/d_has_own.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
```

Comment:
To simplify filenames a bit, can you rename this file to `has.js`? (will need to update any `require` statements)